packages/store/src/secure-store/index.ts

```diff
@@ -4,14 +4,59 @@ import { isAppStudio, errorString } from '../utils';
 import { DummyStore } from './dummy-store';
 import { KeytarStore } from './keytar-store';
 import type { SecureStore } from './types';
+import { join, dirname } from 'path';
+import { homedir } from 'os';
+import { default as fs } from 'fs';
+declare function __non_webpack_require__(m: string): any;
+
+function getKeytarPaths(insiders: boolean): string[] {
+    const vscodeRootPath = insiders ? '.vscode-insiders' : '.vscode';
+    const vscodeExtensionsPath = join(homedir(), vscodeRootPath, 'extensions/');
+    const AppMFoldersVscode =
+        fs
+            .readdirSync(vscodeExtensionsPath)
+            .filter((fn) => fn.startsWith('sapse.sap-ux-application-modeler-extension')) ?? [];
```

Comment:
Should we not allow the consumer to pass the fallback extension rather than hard-coding?

Reply:
We have various consumers e.g. Fiori generator, Adp generator and ui5 tooling modules that need this fallback in certain cases, hence the reason to hardcode it centrally. AppM hosts the SAP System view and should always be installed in Fiori tools setup.

Reply:
Agreed however, this is an internal tools suite extension id and as such may be subject to change. Would it not be more robust to allow the consumer to pass the extension id as the consumer will be able to determine that at runtime.